### ISSUE TITLE
Parsing error: Unexpected token <

### ISSUE THREAD

**Issue by idleberg** [sveltejs/eslint-plugin-svelte3#58]
I'm trying to use `eslint` from the command-line to lint my `*.svelte` files. However, I'm getting the following error.

```sh
/src/App.svelte
  1:1  error  Parsing error: Unexpected token <
```

**Example:**

`.eslintrc.js` (copied from the example in `README.md`)

```js
module.exports = {
  parserOptions: {
    ecmaVersion: 2019,
    sourceType: 'module'
  },
  env: {
    es6: true,
    browser: true
  },
  plugins: [
    'svelte3'
  ],
  overrides: [
    {
      files: ['**/*.svelte'],
      processor: 'svelte3/svelte3'
    }
  ],
  rules: {
    // ...
  },
  settings: {
    // ...
  }
};
```

`Ap.svelte`

```html
<script>
  export let name;
</script>
```

ESLint command

`eslint "src/**/*.{js,svelte}"`

**Setup:**

macOS v10.14.6
Node v12.14.1
eslint 6.8.0
eslint-plugin-svelte3 v2.7.3

**Comment by Conduitry** [comment]
I can't reproduce this. I've created the two files, installed `eslint`, `eslint-plugin-svelte3`, and `svelte`, and run `npx eslint "src/**/*.{js,svelte}"` and I see a linting warning about an unused prop.

**Comment by idleberg** [comment]
I've omitted a crucial part from my issue that appears to be the reason for the error, I didn't think of it as being important.

My config is actually stored in a subfolder, so the actual command is this:

`eslint --config config/eslint.json "src/**/*.{js,svelte}"`

It would be interesting to know reason for the mismatching results, but I can live with my config living in the root of my project.

**Comment by Conduitry** [comment]
I think it is caused by that. It seems to work when specifying the `overrides` as `*.svelte` and not `**/*.svelte`. I forget why I included the globstar in the sample in the readme, but I don't think it's necessary, and it seems to be causing a problem in this case.

**Comment by Conduitry** [comment]
I've changed the readme to just have `*.svelte`.

**Comment by idleberg** [comment]
Thank you, that fixes my problem. Please close the issue if you consider the case closed!
